runbot_pylint/__openerp__.py

```diff
@@ -0,0 +1,25 @@
+# -*- encoding: utf-8 -*-
+##############################################################
+#    Module Writen For Odoo, Open Source Management Solution
+#
+#    Copyright (c) 2011 Vauxoo - http://www.vauxoo.com
+#    All Rights Reserved.
+#    info Vauxoo (info@vauxoo.com)
+#    coded by: moylop260@vauxoo.com
+############################################################################
+
+{
+    'name': 'Runbot Pylint',
+    'category': 'Website',
+    'summary': 'Runbot',
+    'version': '1.0',
+    'author': 'Vauxoo, Odoo Community Association (OCA)',
+    'depends': ['runbot'],
+    'external_dependencies': {
+        'bin': ['pylint'],
```

Comment:
I'd rather this be python instead of bin.
Pylint is a python dependency and can be installed via pip or system packager, either way:

```
pip2 list | grep pylint
pylint (1.4.4)
```
```python
import pylint
assert pylint
```